### ISSUE TITLE
Non-player Staff List

### ISSUE THREAD

**Issue by Dermotron** [archibalduk/TransferTool#11]
The CMUpdater has this done very well but there are no job titles. The TransferTool exports the Non-Players but doesn't include their titles or attributes. Would it be possible to create a export option for non-players. It would essentially include all the CMUpdater exports plus clubs, job titles. On the types page the Preferred formation is very important but the player models aren't at all (I noticed CMUpdater errors when re-importing so I just clear all the fields when using it). zozoa's Database Exporter has all the required fields when exported and players removed. Not sure how to handle player & non-player. They are such a small group they could be left out if needs be

**Comment by MScientistCM** [comment]
@Dermotron  I just tested and the CMUpdater is already exporting titles, attributes and preferred formations, unless im missing something? Can you recheck this in the CMUpdater and let me know?

**Comment by Dermotron** [comment]
It does but has no club data

**Comment by MScientistCM** [comment]
> 
> 
> It does but has no club data, that's what I meant to write the first time

@Dermotron do you mean the club name of the staff? For me it CMUpdater exports with club name correctly

**Comment by Dermotron** [comment]
Non-Player Attributes (Updater Format) is the file I am exporting from CMUpdater v0.0.5 (I don't think there is a newer version)

**Comment by Dermotron** [comment]
It would just be nice to edit some of this stuff and put it back into the DB e.g. out of 21980 staff there are 4737 with no CA set

**Comment by MScientistCM** [comment]
I am using the other two Non-Player export options, they contain the club name. I think you will need a club rep column too. I will have a look.

**Comment by Dermotron** [comment]
Those can't be re-imported though can they? I'm ok with rep not being there.

If they can be re-imported that ignore this thread :D

**Comment by MScientistCM** [comment]
> 
> 
> Those can't be re-imported though can they? I'm ok with rep not being there.
> 
> If they can be re-imported that ignore this thread :D

You can only import the Updater Format. But you can copy and paste the clubs column from the Extended csv into the Updater Format csv, as the staff will be in the same order. Then do your edits in the Updater Format csv, delete the Club column from it and reimport? Does this flow work for you? Or are you willing to reimport the Club column too (i.e. change clubs)?
